omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
```

Comment:
Probably ``anywhere`` where the command can access it... 
Or maybe rather say

``does not have to be placed in the folder from which the omero commands are run``